source/kraken/kraken_zmq.cpp

```diff
@@ -127,8 +127,10 @@ int main(int argn, char** argv) {
 
     const navitia::Metrics metrics(conf.metrics_binding(), conf.instance_name());
 
-    threads.create_thread(
-        [&data_manager, conf, &metrics] { return navitia::MaintenanceWorker(data_manager, conf, metrics); });
+    threads.create_thread([&data_manager, conf, &metrics] {
+        navitia::MaintenanceWorker maintenance_worker = navitia::MaintenanceWorker(data_manager, conf, metrics);
+        return maintenance_worker();
```

Comment:
comment for later understanding: The `return` here is useless, the functor works forever.